### ISSUE TITLE
Farmers run themselves out of carrots and don't keep enough to replant.

### ISSUE THREAD

**Issue by spankymcmillan** [TangoTek/TekTopia-Community#393]
**Describe the bug**
Farmers had nearly a full 9x9 field to work with of carrot crops.  They managed to ruin all the crops/eat all the result and not keep enough to plant more. 

**To Reproduce**
1. Get some superstar 1 int farmers (I had 2 with 1 int and 1 with 9 int)
2. Give them a ton of carrots (or be a real go getter like me and till and plant the carrots for them).
3. Wonder how you ended up with 3 farmers and no carrots.

**Expected behavior**
Not consume or plant all of their carrots so they have enough to replant no matter what.

** Server/Location **
What if 2
7078 71 149

**Additional context**
They had tons of carrots... then they had none.  I do live in an Ice Plains biome.  There are rabbits, which I believe will still grief carrot crops.  Perhaps they in some small part of this could have attributed to the ruining of the crops despite me having rarely seen a rabbit near the crops.  However, they should have kept some in their inventory for replanting.  I believe I had this happen with potatoes as well.  But, luckily I had more of those.

**Comment by JKatBat** [comment]
I did not have tonnes of carrots, Now i have none.

**Comment by orangense** [comment]
I was also struggling a little bit with this. I left steak that I cooked in the chest for them, but they preferred the raw carrots and potatoes (maybe because they weren't hungry enough for steak and given the option they chose carrots or potatoes?). I was able to pull the carrots and potatoes out of storage and hold on to them, and try to force them to eat the steak. Once my farmers got better levels, this was not a problem.
